velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -716,6 +716,57 @@ PlanBuilder& PlanBuilder::localMerge(const std::vector<std::string>& keys) {
   return *this;
 }
 
+PlanBuilder& PlanBuilder::expand(
+    const std::vector<std::vector<std::string>>& projectionSets) {
+  std::vector<std::vector<core::TypedExprPtr>> projectSetExprs;
+  projectSetExprs.reserve(projectionSets.size());
+  std::vector<std::string> names;
+  names.reserve(projectionSets[0].size());
+  std::vector<std::shared_ptr<const Type>> types;
+  types.reserve(projectionSets[0].size());
+  std::string groupIdPrefix = "group_id_";
+  int grouIdColCount = 0;
+  for (int i = 0; i < projectionSets[0].size(); ++i) {
+    for (int j = 0; j < projectionSets.size(); ++j) {
+      if (projectionSets[j][i] != "") {
+        if (planNode_->outputType()->containsChild(projectionSets[j][i])) {
+          names.push_back(projectionSets[j][i]);
+          types.push_back(
+              field(planNode_->outputType(), projectionSets[j][i])->type());
+        } else {
+          names.push_back(groupIdPrefix + std::to_string(grouIdColCount++));
+          types.push_back(BIGINT());
```

Comment:
Notice that BIGINT is not an INTEGER, but the comments for Expand operator say that only integer values are allowed.

Reply:
The grouping_id column is long type not int in spark Expand operator.